profile/2019-03-01-hybrid.json

```diff
@@ -0,0 +1,236 @@
+{
+        "info":{
+    
+            "name":"2019-03-01-hybrid",
+    
+            "description":"Profile definition targeted for hybrid applications that could run on azure stack general availability version and azure cloud for 1903."
+    
+        },
+    "resource-manager": {
+
+        "microsoft.authorization": {
+
+            "2016-09-01": [
+                    "locks"
+            ],
+
+            "2016-12-01": [
+                    "policyDefinitions",
+                    "policyAssignments"
+            ],
+
+            "2015-07-01": [
+                    "operations",
+                    "permissions",
+                    "roleAssignments",
+                    "roleDefinitions"
+            ],
+
+            "2015-07-01-preview":[
+                    "providerOperations"
+            ]
+        },
+
+        "microsoft.compute": {
+
+            "2017-12-01":[
+                    "availabilitySets",
+                    "images",
+                    "locations",
+                    "locations/publishers",
+                    "locations/operations",
+                    "locations/usages",
+                    "locations/vmSizes",
+                    "operations",
+                    "virtualMachines",
+                    "virtualMachines/extensions",
+                    "virtualMachineScaleSets",
+                    "virtualMachineScaleSets/extensions",
+                    "virtualmachineScaleSets/networkInterfaces",
+                    "virtualMachineScaleSets/virtualMachines",
+                    "virtualMachineScaleSets/virtualMachines/networkInterfaces"
+            ],
+
+            "2017-03-30":[
+                    "disks",
+                    "locations/diskoperations",
+                    "snapshots"
+            ]
+        },
+
+        "microsoft.eventhub": {
+
+            "2018-01-01-preview":[
+                    "namespaces",
+                    "checkNameAvailability",
+                    "sku",
+                    "operations"
+            ],
+
+            "2015-08-01":[
+                    "checkNamespaceAvailability"
+            ]
+        },
+
+        "microsoft.insights": {
+            "2018-01-01": [
+                    "metricDefinitions",
+                    "metrics"
+            ],
+
+            "2017-05-01-preview": [
+                    "diagnosticSettings",
+                    "diagnosticSettingCategories"
+            ],
+
+            "2017-03-01-preview": [
```

Comment:
api version for the event* resource types would be 2015-04-01, -preview is just being used for internal UX purposes

Reply:
Does that mean it shouldn't be included in the generated SDK profiles?

Reply:
Yes, the insights team does not want the 2017* api version in the SDKs or tools. The api version got consumed, just for easy porting of the Ux, it did not add any new features in that api-version